book/book.toml

```diff
@@ -18,11 +18,15 @@ sidebar-width = "250px"
 
```

Comment:
Per @aacebo's request, do we need to update author?

Reply:
I updated the author to Microsoft 📦